web/src/games/kriegspiel/index.ts

```diff
@@ -0,0 +1,25 @@
+const Thumbnail = require('./media/thumbnail.png?lqip-colors');
+import { GameMode } from 'gamesShared/definitions/mode';
+import { IGameDef, IGameStatus } from 'gamesShared/definitions/game';
+import instructions from './instructions.md';
+
+export const kriegspielGameDef: IGameDef = {
+  code: 'kriegspiel',
+  name: "Guy Debord's Kriegspiel",
```

Comment:
We generally dont allow using the original game name for copyright reasons. However, this game doesn't seem to be historic and not commercialized anymore, so we might be OK leaving the game name as is, and if someone complains we can change it later. https://en.wikipedia.org/wiki/A_Game_of_War

Reply:
(no action needed)